Modules/SubModule_Import/xASL_bids_BIDSifyCheckTimeEncoded.m

```diff
@@ -62,14 +62,26 @@
         bTimeEncodedFME = ~isempty(regexp(jsonOut.SeriesDescription,'(Encoded_Images_Had)\d\d(_)\d\d(_TIs_)\d\d(_TEs)', 'once'));
         if bTimeEncodedFME
             startDetails = regexp(jsonOut.SeriesDescription,'\d\d(_)\d\d(_TIs_)\d\d(_TEs)', 'once');
-            jsonOut.HadamardType = xASL_str2num(jsonOut.SeriesDescription(startDetails:startDetails+1));
-            jsonOut.HadamardNumberPLD = xASL_str2num(jsonOut.SeriesDescription(startDetails+3:startDetails+4));
-            jsonOut.HadamardNumberTE = xASL_str2num(jsonOut.SeriesDescription(startDetails+10:startDetails+11));
+            jsonOut.TimeEncodedMatrixSize = xASL_str2num(jsonOut.SeriesDescription(startDetails:startDetails+1));
+            jsonOut.TimeEncodedNumberPLD = xASL_str2num(jsonOut.SeriesDescription(startDetails+3:startDetails+4));
+            jsonOut.TimeEncodedNumberTE = xASL_str2num(jsonOut.SeriesDescription(startDetails+10:startDetails+11));
```

Comment:
@BeatrizPadrela: Please check if these are the correct fieldnames.

Reply:
Will do now :)